docs/developers/CwlParsing.md

```diff
@@ -0,0 +1,40 @@
+# How CWL is Parsed
+
+The process of parsing CWL runs approximately like this:
+
+1. Produce a canonical one-file representation of the CWL as a JSON object in memory 
```

Comment:
[ToL] Since this is a developer doc, it might be nice to mention the top-level function or class that handles each of these operations